proto/elys/amm/params.proto

```diff
@@ -14,4 +14,5 @@ message Params {
     (gogoproto.nullable) = false
   ];
   uint64 slippage_track_duration = 2; // default 1 week: 604,800
+  bool enable_usdc_paired_pool_only = 3;
```

Comment:
@cryptokage1996 can we rename this param to `enable_base_currency_paired_pool_only` instead as we might replace the base currency from `usdc` to any other stable coin in the future?